src/libUtils/Logger.h

```diff
@@ -59,7 +59,7 @@ class Logger {
   std::string m_fileName;
   std::ofstream m_logFile;
   unsigned int m_seqNum;
```

Comment:
It seems like `clang-tidy` doesn't necessarily complain, since native-type members like this remain unmodified.

Reply:
`Initialize every variable not being initialized by default constructor`